src/components/elements/table/header.js

```diff
@@ -1,21 +1,53 @@
+import { Component } from 'react'
 import cx from 'classnames'
 
-export default ({ api, property }) => {
-  if (!property.columns) {
-    return <thead/>
+export default class Header extends Component {
+  constructor(props) {
+    super()
+
+    this.state = {
+      sortedColumn: null,
+      direction: null,
+    }
+  }
+
+  render() {
+    if (!this.props.columns) {
+      return <thead />
+    }
+    return (
+      <thead>
+        <tr className='ct-table-header-row'>
+          {this.props.columns.map((column, columnIndex) => {
```

Comment:
Thanks, changed. I haven't quite got the hang of when you need them and when you don't.